### PR TITLE
Show Guggy GIFs attachments

### PR DESCRIPTION
**Pull request by luciofm**
depends on https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK/pull/102

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/chatroom/viewmodel/ViewModelMapper.kt [hunk 1]
```diff
@@ -303,4 +307,16 @@ class ViewModelMapper @Inject constructor(private val context: Context,
             })
         }
     }
-}
\ No newline at end of file
+}
+
+internal val FileAttachment.attachmentTitle: String
```
Comment (leonardoaramaki) [inline app/src/main/java/chat/rocket/android/chatroom/viewmodel/ViewModelMapper.kt]: I'm not sure if it makes sense using the `internal` modifier on any class from the app project.

Comment (luciofm) [reply app/src/main/java/chat/rocket/android/chatroom/viewmodel/ViewModelMapper.kt]: True...

File: dependencies.gradle [hunk 2]
```diff
@@ -86,7 +86,7 @@ ext {
         kotshiApi              : "se.ansman.kotshi:api:${versions.kotshi}",
         kotshiCompiler         : "se.ansman.kotshi:compiler:${versions.kotshi}",
 
-        frescoImageViewer      : "com.github.stfalcon:frescoimageviewer:${versions.frescoImageViewer}",
+        frescoImageViewer      : "com.github.luciofm:FrescoImageViewer:${versions.frescoImageViewer}",
```
Comment (leonardoaramaki) [inline dependencies.gradle:89]: Uhmm. I don't trust very much on libraries coming from that github... :laughing:

**Review by leonardoaramaki — APPROVED** [review]

File: dependencies.gradle (hunk 2 above)
Comment (luciofm) [reply dependencies.gradle:89]: Just waiting Picasso3 to get rid of it 😄

**Pull request merged**